application/schemas.py

```diff
@@ -0,0 +1,36 @@
+from pydantic import BaseModel
+from datetime import datetime
+
+
+class CityBase(BaseModel):
+    name: str
+    description: str = None
```

Comment:
The `description` field in `CityBase` is not present in the `City` model in `models.py`. Ensure that the schema and model definitions are aligned to avoid errors.